registry.py

```diff
@@ -188,7 +188,7 @@ class Registry:
 
     # this is required for proper digest processing
     HEADERS = {"Accept":
-               "application/vnd.docker.distribution.manifest.v2+json"}
+               "application/vnd.docker.distribution.manifest.v2+json,application/vnd.oci.image.manifest.v1+json"}
```

Comment:
Maybe a comment which is for Docker, which is for Podman and that without the other there Podman images keep lingering?